src/main/java/cloud/fogbow/ras/core/cloudconnector/LocalCloudConnector.java

```diff
@@ -265,6 +265,44 @@ public void deleteSecurityRule(String securityRuleId, SystemUser systemUser) thr
         }
     }
 
+    @Override
+    public void pauselInstance(Order order) throws FogbowException {
+        LOGGER.debug(String.format(Messages.Log.MAPPING_USER_OP_S, DELETE_INSTANCE_OPERATION, order));
+        CloudUser cloudUser = this.mapperPlugin.map(order.getSystemUser());
+        LOGGER.debug(String.format(Messages.Log.MAPPED_USER_S, cloudUser));
+
+        String response = null;
+        try {
+            doDeleteInstance(order, cloudUser);
```

Comment:
Change the name of the method and add a trivial implementation for the time being.